src/msw/srchctrl.cpp

```diff
@@ -0,0 +1,448 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        src/msw/srchctrl.cpp
+// Purpose:     wxSearchCtrl implementation - native
+// Author:      Kettab Ali
+// Created:     2019-12-23
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+// For compilers that support precompilation, includes "wx.h".
+#include <wx/wxprec.h>
+
+#ifdef __BORLANDC__
+    #pragma hdrstop
+#endif
+
+#if wxUSE_SEARCHCTRL
```

Comment:
```suggestion
#if wxUSE_SEARCHCTRL && wxUSE_GRAPHICS_CONTEXT
```
Otherwise this is not going to compile due to the use of `wxGCDC`.